api/models.go

```diff
@@ -17,6 +17,11 @@ type ListResponse struct {
 	Total int64 `json:"total"`
 }
 
+type NameRequest struct {
```

Comment:
Rename to `NameFilter` as it is for filter by name field and not a request for Names.

Reply:
Why? I use it for add / update name and not for filter

Reply:
but you don't request `Names`. `Name` is not an entity (R in REST). It's a resource attribute, a field.

Reply:
renamed